scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
+as well as the script to perform the conversion.
+
+Rules:
```

Comment:
How about "Dictionary entries must follow the following rules:" or so?

Reply:
I like this, thanks!